reference/7.0/Microsoft.PowerShell.Core/About/about_Calculated_Properties.md

```diff
@@ -0,0 +1,506 @@
+---
+Locale: en-US
+ms.date: 08/07/2020
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_calculated_properties?view=powershell-7&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about_Calculated_Properties
+---
+# About Calculated Properties
+
+## Short Description
+
+Calculated properties allow you to dynamically add properties and formatting to
+objects in the pipeline.
+
+## Long Description
+
+PowerShell provides several cmdlets that operate on properties of objects.
+PowerShell provide a method to create calculated properties. Calculated
+properties are dynamically defined at runtime. Calculated properties can be use
+to dynamically create new properties that are added to the output object or
+provide custom formatting for an existing property.
+
+Each cmdlet that supports calculated properties has specific set of attributes
+that describe the property. The attributes define the name of the property, an
+expression to calculate the value, and optional formatting information. A
+calculated property is defined using a hashtable containing these attributes as
+key-value pairs.
+
+## Supported cmdlets
+
+The following cmdlets support calculated properties using the listed property
+attributes names as keys in the hashtable.
+
+> [!NOTE]
+> The `expression` attribute can be a script block instead of a hashtable. For
+> more information, see the [Notes](#notes) section. For descriptions of these
+> attibutes, see [Attribute definitions](#attribute-definitions).
+
+- `Compare-Object`
+  - `expression`
+
+- `ConvertTo-Html`
+  - `name`/`label` - optional
+  - `expression`
+  - `width` - optional
+  - `alignment` - optional
+
+- `Format-Custom`
+  - `expression`
+  - `depth` - optional
+
+- `Format-List`
+  - `name`/`label` - optional
+  - `expression`
+  - `formatstring` - optional
+
+  This same set of attributes also apply to the `-GroupBy` properties for all
```

Comment:
applies

Reply:
I don't think this should be applies? I could be wrong but I think apply is _applied_ correctly ;)

Reply:
It is applies - "set" is a singular noun.